React/CoreModules/RCTExceptionsManager.mm

```diff
@@ -138,9 +138,9 @@ - (void)reportFatal:(NSString *)message stack:(NSArray<NSDictionary *> *)stack e
   }
 }
 
-- (void)reportEarlyJsException:(std::string)errorMap
+- (void)reportEarlyJsException:(NSString *)errorMap
 {
-  NSString *errprStr = [NSString stringWithUTF8String:errorMap.c_str()];
+  NSString *errprStr = errorMap;
```

Comment:
would these also need revert @dmitryrykun ? just scanning this after seeing your cmoment and the seem similar

Reply:
Yeah, I meant all the changes in RCTExceptionsManager.